models/iaf_cond_alpha_mc_fixedca.h

```diff
@@ -0,0 +1,663 @@
+/*
+ *  iaf_cond_alpha_mc_fixedca.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef IAF_COND_ALPHA_MC_FIXEDCA_H
+#define IAF_COND_ALPHA_MC_FIXEDCA_H
+
+// Generated includes:
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// C++ includes:
+#include <vector>
+
+// C includes:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "recordables_map.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+// Includes from sli:
+#include "dictdatum.h"
+#include "name.h"
+
+/* BeginDocumentation
+Name: iaf_cond_alpha_mc_fixedca - PROTOTYPE Multi-compartment conductance-based leaky
+integrate-and-fire
+neuron model with calcium spike at distal compartment modeled using threshold triggered fixed
+waveform.
```

Comment:
The description following the - sign is used in documentation table of contents and should therefore be short, preferably no more than 60 characters. Could you shorten it?